client-app/src/examples/contactMobile/details/ContactDetails.ts

```diff
@@ -0,0 +1,145 @@
+import {XH, hoistCmp, uses} from '@xh/hoist/core';
+import {box, div, img, p, filler} from '@xh/hoist/cmp/layout';
+import {formField} from '@xh/hoist/mobile/cmp/form';
+import {form} from '@xh/hoist/cmp/form';
+import {panel} from '@xh/hoist/mobile/cmp/panel';
+import {select, textArea, textInput} from '@xh/hoist/mobile/cmp/input';
+import {button} from '@xh/hoist/mobile/cmp/button';
+import {toolbar} from '@xh/hoist/mobile/cmp/toolbar';
+import {Icon} from '@xh/hoist/icon';
+import {isEmpty, get} from 'lodash';
+
+import ContactDetailsModel from './ContactDetailsModel';
+import './ContactDetails.scss';
+
+export const contactDetails = hoistCmp.factory({
+    model: uses(ContactDetailsModel),
+    render({model}) {
+        const {currentRecord, visible} = model;
+
+        return panel({
+            className: `tb-mobile-contact-details-panel${visible ? '--details-visible' : ''}`,
+            flex: 'none',
+            position: 'absolute',
+            title: currentRecord?.data.name,
+            headerItems: [button({icon: Icon.close(), onClick: () => model.clearCurrentRecord()})],
+            item: currentRecord ? contactProfile() : null,
+            bbar: bbar()
+        });
+    }
+});
+
+const contactProfile = hoistCmp.factory({
```

Comment:
All the definitions below here are another instance where they LOOK the same as the desktop counterparts, but use MOBILE versions of components.